src/Tizen.Nlp/Tizen.Nlp/NlpCommon.cs

```diff
@@ -0,0 +1,174 @@
+/*
+* Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Collections;
+using Tizen;
+using System.Threading.Tasks;
+using Tizen.Applications;
+using RpcPort.Message.Proxy;
+using Tizen.Applications.RPCPort;
+
+namespace Tizen.Nlp
+{
+    
+    /// <summary>
+    /// This class contains the methods and inner class related to the NLP processing.
+    /// </summary>
+    /// <since_tizen> 5 </since_tizen>
+    public class NltkClass
```

Comment:
Please read the document about naming and layout of class. 
https://chodounsky.net/2012/12/12/csharp-naming-and-layout-guidelines/